Tribute Page/index.html

```diff
@@ -15,50 +15,62 @@ <h3>
         Early Life
       </h3>
         <p>
-          Elon Reeve Musk was born in Pretotia, the capital of South Africa, on June 28, 1971. After spending his childhood with his brother Kimbal and sister Tosca, at the age of 10, he developed an interest in computers. He was so much into computers that after
-          teaching himself to program, he sold his first software(a computer game) called 'Blaster'; he was 12 years old then.
+          <strong><i>Elon Reeve Musk</i></strong> was born in Pretotia, the capital of South Africa, on <u>June 28, 1971</u>. After spending his childhood with his brother Kimbal and sister Tosca, at the age of 10, he developed an interest in computers. He was so much into computers that after
+          teaching himself to program, he sold his first software(a computer game) called '<i><b><u>Blaster</u></b></i>'; he was 12 years old then.
         </p>
       </div>
       <div id="education">
         <h3>
-        Education
+        <strong><i>Education:</i></strong>
       </h3>
         <p>
-          Musk attended the English-speaking Waterkloof House Preparatory School, and later graduated from Pretoria Boys High School.<br/> At 17, Musk moved to Canada to avoid serving in the South African military, whose main duty in the late 1980s was
+          <i>Musk</i> attended the English-speaking Waterkloof House Preparatory School, and later graduated from Pretoria Boys High School.<br/> At 17, Musk moved to Canada to avoid serving in the South African military, whose main duty in the late 1980s was
           enforcing apartheid. SAfter emigrating to Canada, Musk enrolled in Queen's University in Kingston, Ontario. He studied physics there. <br/> Also he graduated with bachelor's degree in economics from the Wharton school of business.<br/> At 24,
-          he went to pursue his PhD. in applied Physics at Stanford University.
+          he went to pursue his PhD. in applied Physics at <a href="https://www.stanford.edu">Stanford University</a>.
         </p>
       </div>
       <div id="boe">
         <h3>
-        The birth of an Entrepeneur
+        </i><strong>The birth of an Entrepeneur</i></strong>

```

Comment:
```suggestion
        <i><strong>The birth of an Entrepeneur</strong></i>
```
Please fix the <italics> and the <strong> tags along with their position.